docs/environment_creation_from_scratch.md

```diff
@@ -1,24 +1,21 @@
 # Creating a new environment from scratch
 
-TODO: Update this once splitting out secrets and app roles is done, add instructions for sorting those and things to update in meta.
-
-When setting up a new environment;
+When setting up a new environment:
 1. DNS records will need to be created by DLUHC for certificate validation and to point at the cloudfront distribution and app load balancer
 1. Secret values will need to be filled in manually
-1. Need to set up deployments etc.
-1. Need to ensure anything that needs adding to the meta module (e.g. for roles access to ecr) is done, since some of that is currently manual
+1. The meta account will need updating with additional roles access to the ecr, since this is currently manual
+1. A new deployment pipeline will need to be setup
 
 ## The process
 
 ### Run an initial apply
 
-```terraform apply -target="module.networking" -target="module.front_door" -var="initial_create=true"```
+```terraform apply -target="module.networking" -target="module.front_door" -target="module.certificates" --target="module.application_roles" --target="module.application_secrets" var="initial_create=true"```
```

Comment:
Do you need to specify the certificates module? I thought terraform automatically picked up that the certificates were a dependency of the front_door module and so created them.

Reply:
IIRC - when I was deploying the dev env for DNS (using `initial_create = true`), the `tf apply` would only create the load balancer certificate, even though the output from it's plan said it would create the cloudfront one as well. 

I reckon this is because of the line below in the `cloudfront.tf` file
```
acm_certificate_arn  = var.initial_create ? null : var.cloudfront_certificate_arn
```

So with `initial_create = true`, the certificate arn isn't actually needed for the various bits of the `front_door` module to be built, but the `front_door` module takes this as an input parameter, which is possibly why the plan is misleading.

Nonetheless, we still need both certificates to be made to ensure that we can give all DNS details to DLUHC, and adding this to the command was a way to ensure it 😀

Reply:
I remember it working without when setting up the prod environment, because terraform considers both to be dependencies of the module even though neither is then actually used (as the load balancer one is only attached in a listener resource that is not created when initial_create is true). So I'm surprised if it didn't work for dev. 

Nonetheless the certificates module doesn't have other things in it that we'd want not to create yet, and I'm assuming copy-paste rather than typing, so 🤷‍♀️

Reply:
Hmm very odd. I'm unsure if it's because of some of the refactoring work we did for `[CLDC-2811](https://dluhcdigital.atlassian.net/browse/CLDC-2811)` that made me see this on dev. 

I've removed `-target=module.certificates` from the command for now, and will add an instruction to try this target if the certificates haven't been made as expected as part of [CLDC-2763](https://dluhcdigital.atlassian.net/browse/CLDC-2763)